### PR TITLE
Update technical-reference-details-about-encryption.md

### PR DESCRIPTION
**Pull request by msbemba**
made changes to Support for TLS 1.0 and 1.1 deprecation

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/10188

File: microsoft-365/compliance/technical-reference-details-about-encryption.md [hunk 1]
```diff
@@ -55,7 +55,7 @@ TLS version 1.3 (TLS 1.3) is supported by some of the services.
   
 ## Support for TLS 1.0 and 1.1 deprecation
 
-Office 365 stopped supporting TLS 1.0 and 1.1 on October 31, 2018. We have completed disabling TLS 1.0 and 1.1 in GCC High and DoD environments. We began disabling TLS 1.0 and 1.1 for Worldwide and GCC environments beginning on October 15, 2020 and will continue with roll-out over the next weeks and months.
+Office 365 stopped supporting TLS 1.0 and 1.1 on October 31, 2018. We have completed disabling TLS 1.0 and 1.1 in GCC High, Worldwide and DoD environments. For Microsoft 365 operated by 21 Vianet, TLS 1.0/1.1 will be disabled on June 30, 2023.
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/technical-reference-details-about-encryption.md]:
```suggestion
Office 365 stopped supporting TLS 1.0 and 1.1 on October 31, 2018. We have completed disabling TLS 1.0 and 1.1 in GCC High, Worldwide, and DoD environments. For Microsoft 365 operated by 21 Vianet, TLS 1.0 and 1.1 will be disabled on June 30, 2023. 
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by yogkumgit** [comment]
@JohanFreelancer9

**Comment by yogkumgit** [comment]
@KCCross Please merge this PR. Thanks

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
Closing as per the comment in #10188

**Pull request closed without merging**